.github/workflows/conformance-tests.yaml

```diff
@@ -33,12 +33,27 @@ jobs:
           - ${{ inputs.image-variant }}
         version:
           - ${{ inputs.version }}
+        version-files:
```

Comment:
This is a shift in behavior. Previously, we ran min/max versions on an async schedule (ie not on PRs) and it seems we are shifting that to run this on PRs. This isn't bad per se, I just wanted to call it out. What are we trying to achieve by running min/max on PRs, that running these nightly did not achieve? Or, if we didn't run the min/max for conformance tests nightly, perhaps we should do that instead?